multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -103,21 +103,41 @@ const toOracleOfferId = (idx: number) => `oracle${idx + 1}-accept`;
 
 test.serial('oracles accept', async t => {
   const { oracleWds, retryUntilCondition, vstorageClient, wallets } = t.context;
+  const brands = await vstorageClient.queryData('published.agoricNames.brand');
+  const { Invitation } = Object.fromEntries(brands);
 
-  const instances = await vstorageClient.queryData(
-    'published.agoricNames.instance',
-  );
-  const instance = fromEntries(instances)[contractName];
+  const description = 'oracle operator invitation';
+
+  // ensure we have an unused (or used) oracle invitation in each purse
+  let hasAccepted = false;
+  for (const name of keys(oracleMnemonics)) {
```

Comment:
There's a lot going on in this loop.

If we reify the oracle operators as objects somewhat like [makeOracleOperator](https://github.com/Agoric/agoric-sdk/blob/d35659b6ab227f11f952d864d38add58dd464825/packages/fast-usdc/test/fast-usdc.contract.test.ts#L208) or [makeAgoricWatcher](https://github.com/Agoric/agoric-sdk/blob/74d293b9fae15081b92e3876beba7ed6278e4b39/packages/fast-usdc/test/quickSend-tx.test.ts#L151), we can make this read much more straightforwardly:

```suggestion
  for (const op of oracleOperators) {
    const status = await oracle.invitationCheck(); // returns the invitation or `used` if used;
    t.log({ name: op.name, status });
    t.truthy(status);
  }
```